main/parsers.h

```diff
@@ -51,6 +51,7 @@
 	PythonParser, \
 	RexxParser, \
 	RubyParser, \
+	RestParser, \
```

Comment:
That would apply to classes and method names then, too?!

Maybe `ReST` could be used instead?